### ISSUE TITLE
rspconfig setting hostname broken for last 2 day (PR #4326)

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#4378]
Running my UT autoamtion, I find that setting hostname is broken, caused by https://github.com/xcat2/xcat-core/pull/4326.....

```
------START::rspconfig_set_hostname::Time:Mon Nov 27 09:47:00 2017------

RUN:rspconfig mid05tor12cn16 hostname=mid05tor12cn16-UTset [Mon Nov 27 09:47:00 2017]
ElapsedTime:1 sec
RETURN rc = 1
OUTPUT:
mid05tor12cn16: Error: AttributeError("Attribute 'parameter_list' not defined.",)
Mon Nov 27 09:47:01 2017 mid05tor12cn16: [openbmc_debug] curl -k -c cjar -H "Content-Type: application/json" -d '{ "data": ["root", "xxxxxx"] }' https://172.11.139.16/login
Mon Nov 27 09:47:01 2017 mid05tor12cn16: [openbmc_debug] login_response 200 OK
Mon Nov 27 09:47:01 2017 mid05tor12cn16: [openbmc_debug] curl -k -b cjar -X PUT -H "Content-Type: application/json" -d '{"data":"mid05tor12cn16-UTset"}' https://172.11.139.16/xyz/openbmc_project/network
Mon Nov 27 09:47:01 2017 mid05tor12cn16: [openbmc_debug] rspconfig_set_response 500 Internal Server Error
CHECK:rc == 0	[Failed]

------END::rspconfig_set_hostname::Failed::Time:Mon Nov 27 09:47:01 2017 ::Duration::1 sec------
```
It looks like the endpoint got deleted in the last code changes to rspconfig... ..

The UT scripts work and show the correct HostName endpoint. 

```
[root@briggs01 Network]# ./set_hostname.sh f6u18-bmc
++ date
Mon Nov 27 09:25:29 EST 2017
++ curl -c cjar -b cjar -k -H 'Content-Type: application/json' -X PUT -d '{"data": "f6u18-bmc" }' https://mid05tor12cn18-bmc//xyz/openbmc_project/network/config/attr/HostName
{
  "data": null,
  "message": "200 OK",
  "status": "ok"
}
```

**Comment by whowutwut** [comment]
Testcases added with #4379

**Comment by gurevichmark** [comment]
Fixed by PR #4380 
Seems that #4326 removed the url path for hostname setting of `/config/attr/HostName`

**Comment by whowutwut** [comment]
Closing this issue out, if we do not see it occur on automation #4370, then should be fixed.
